### ISSUE TITLE
Create Developer Mailing List

### ISSUE THREAD

**Issue by simnh** [oemof/organisation#37]
We discussed it would be great to have a developer mailing list.

**Comment by uvchik** [comment]
Did you discuss what would be on github and what would be on the mailing list? At the moment oemof-organisation is kind of our open mailing-list.

https://posteo.de/hilfe/gibt-es-mailinglisten-bei-posteo

**Comment by simnh** [comment]
Yes you are right, but I think there is a difference still. In particular as not everything is to be openly discussed I think.

**Comment by uvchik** [comment]
We could use `developer@oemof.org`. I would activate this name but anybody else should create the list.

@oemof/oemof-developer-group Does anybody has a posteo account and can ask for a list? All information can be found [here](https://posteo.de/hilfe/gibt-es-mailinglisten-bei-posteo).

**Comment by ckaldemeyer** [comment]
> @oemof/oemof-developer-group Does anybody has a posteo account and can ask for a list? All information can be found [here](https://posteo.de/hilfe/gibt-es-mailinglisten-bei-posteo).

I don't have a posteo-account..

**Comment by p-snft** [comment]
Please note that there is certain effort to change the decision for a mailing list provider, once it is taken. Especially, users would have to re-register. Also, I do not believe using developer@oemof.org in combination with posteo would be simple.

Maybe, this is coupled to the discussion on whether we should move the website to another provider. (Many but WP.com, have mailing lists included.)

**Comment by uvchik** [comment]
> Please note that there is certain effort to change the decision for a mailing list provider, once it is taken. Especially, users would have to re-register.

The register process is quit easy and the mailing list works good. If you have a better solution go ahead.  A big pro is that posteo does not collect our data. @simnh asked for a mailing list and I think the decision about a new provider will take its time, so I would not couple the decission.

> Also, I do not believe using [developer@oemof.org](mailto:developer@oemof.org) in combination with posteo would be simple.

We already use this combination contact@oemof.org -> oemof@lists.posteo.de

We could use the same list but I thought it might be confusing, so I suggested to use a new combination e.g. dev@oemof.org -> oemof-dev@lists.posteo.de.

But of course the easiest way to start would be to forward dev@oemof.org to the existing list because the traffic there is very low and everybody is free to filter (filter whether the mail comes from contact@oemof.org or from dev@oemof.org). This might be the easiest interim solution.

**Comment by ckaldemeyer** [comment]
> > Please note that there is certain effort to change the decision for a mailing list provider, once it is taken. Especially, users would have to re-register.
> 
> The register process is quit easy and the mailing list works good. If you have a better solution go ahead. A big pro is that posteo does not collect our data. @simnh asked for a mailing list and I think the decision about a new provider will take its time, so I would not couple the decission.
> 
> > Also, I do not believe using [developer@oemof.org](mailto:developer@oemof.org) in combination with posteo would be simple.
> 
> We already use this combination [contact@oemof.org](mailto:contact@oemof.org) -> [oemof@lists.posteo.de](mailto:oemof@lists.posteo.de)
> 
> We could use the same list but I thought it might be confusing, so I suggested to use a new combination e.g. [dev@oemof.org](mailto:dev@oemof.org) -> [oemof-dev@lists.posteo.de](mailto:oemof-dev@lists.posteo.de).
> 
> But of course the easiest way to start would be to forward [dev@oemof.org](mailto:dev@oemof.org) to the existing list because the traffic there is very low and everybody is free to filter (filter whether the mail comes from [contact@oemof.org](mailto:contact@oemof.org) or from [dev@oemof.org](mailto:dev@oemof.org)). This might be the easiest interim solution.

Sorry for bringing this up once again: what about a google-group?

I myself have concerns about google but almost everybody has an (anonymous) google account and it is one click, self-archiving, has a web-interface, is used by numerous other projects, ...

**Comment by ckaldemeyer** [comment]
I revoke my last post and go with @uvchik !

**Comment by uvchik** [comment]
Le us give it a try. dev@oemof.org is now forwarded to the mailing list and everybody is free to self-subscribe to the list using https://lists.posteo.de/listinfo/oemof.

**Comment by busiing** [comment]
Should we send your post to the list from the last meeting?
